plugin.yaml

```diff
@@ -0,0 +1,69 @@
+##################################################################################
+# Cloudify host pool built in types and plugins definitions.
+##################################################################################
+
+plugins:
+    hostpool:
+        executor: central_deployment_agent
+        source: https://github.com/cloudify-cosmo/cloudify-host-pool-plugin/archive/master.zip
+
+node_types:
+  host_pool_node:
```

Comment:
this will be consistent with the rest of our plugins.
i.e : cloudify.hostpool.nodes.Host